manifests/kustomize/third-party/argo/base/workflow-controller-configmap-patch.yaml

```diff
@@ -39,10 +39,3 @@ data:
         key: secretkey
   executor: |
     imagePullPolicy: IfNotPresent
-    resources:
```

Comment:
@gkcalat while this change makes sense to me, can you double check if the issue described by https://github.com/kubeflow/pipelines/pull/5695 which introduced this resource config still exists? By removing this, would we possibly break again the original issue? If so, would there be a better solution to the original issue other than having this global config?

Reply:
Discussed offline. The change has been verified on full Kubeflow 1.7.0 on GKE 1.24. The same pipeline runs completed successfully.